public/toolkit/styles/molecules/_inputs.scss

```diff
@@ -383,3 +383,27 @@ input.icon-input-field {
   line-height: $icon-input-height;
   height: $icon-input-height;
 }
+
+// Search text field with x icon to clear
+// Javascript must be added to
+$clear-icon-margin: 13.5px;
+$clear-icon-size: 16px;
+.search {
+  display: inline-block;
+  position: relative;
+  width: 100%;
+
+  input {
+    width: 100%;
+  }
+  input.has-value {
+    // Center the icon within the padding. Subtract 1px to account for the border.
+    padding-right: rem-calc(($clear-icon-margin - 1) * 2 + $clear-icon-size);
```

Comment:
Small q about the math here. rem-calc takes a pixel value and returns a rem value, right? And then you can just add a pixel and a rem value together no problem?

Reply:
I wasn't sure if it would work adding pixels plus rems so I just did the math in pixels then did remcalc on the final product

Reply:
Oh you're right, I missed that the whole this is wrapped in rem-calc